Sofa/Component/Collision/Detection/Algorithm/src/sofa/component/collision/detection/algorithm/CollisionPipeline.h

```diff
@@ -57,4 +57,8 @@ class SOFA_COMPONENT_COLLISION_DETECTION_ALGORITHM_API DefaultPipeline : public
     static const int defaultDepthValue;
 };
 
+
+SOFA_DEPRECATED_HEADER("v23.06", "v23.12", "sofa/component/collision/detection/algorithm/CollisionPipeline.h")
+using DefaultPipeline = sofa::component::collision::detection::algorithm::CollisionPipeline;
+
```

Comment:
I dont understand why you remove the deprecation macro and added this ?
People including the good `CollisionPipeline.h` file will have a deprecation warning about using .... `CollisionPipeline.h` 🤔